.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  pull_request:
+
+jobs:
+  dib-lint:
+    name: Run dib-lint
+    runs-on: ubuntu-latest
+
+    env:
+      ELEMENTS_DIR: elements
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+
+      - name: Create & activate venv, install dib-lint
+        shell: bash
+        run: |
+          python -m venv .venv
+          source .venv/bin/activate
+          pip install --upgrade pip
+          pip install diskimage-builder
+
+      - name: Add dummy README.rst where missing
+        shell: bash
+        run: |
+          for d in elements/*; do
+            if [[ -d "$d" && ! -f "$d/README.rst" ]]; then
+              echo "Empty README for dib-lint" > "$d/README.rst"
+            fi
+          done
+
+      - name: Run dib-lint
+        shell: bash
+        run: |
+          source .venv/bin/activate
+          dib-lint
```

Comment:
I'm not sure we need to bother with creating + using a venv in the ephemeral environment

Reply:
fair point. force of habit i guess. removed the venv stuff here